Foundation/NSGeometry.swift

```diff
@@ -459,8 +476,7 @@ public func NSIntegralRectWithOptions(_ aRect: NSRect, _ opts: NSAlignmentOption
     if aRect.size.width.native < 0 {
         width = 0
     }
-    
-
+#if !os(Android) // FIXME!!!
```

Comment:
I’m still arguing with the compiler about exactly what is going on in NSGeiometry.swift re types at the moment a signifiant peice of code is effectively commented out.